kyo-prelude/shared/src/main/scala/kyo/Stream.scala

```diff
@@ -328,25 +329,90 @@ final case class Stream[V, -S](v: Ack < (Emit[Chunk[V]] & S)):
 end Stream
 
 object Stream:
+    @nowarn("msg=anonymous")
+    inline def apply[V, S](inline v: => Ack < (Emit[Chunk[V]] & S)): Stream[V, S] =
+        new Stream[V, S]:
+            def emit: Ack < (Emit[Chunk[V]] & S) = v
 
     private val _empty           = Stream(Stop)
     def empty[V]: Stream[V, Any] = _empty.asInstanceOf[Stream[V, Any]]
 
-    def init[V, S](seq: Seq[V] < S)(using tag: Tag[Emit[Chunk[V]]], frame: Frame): Stream[V, S] =
+    private inline def DefaultChunkSize = 4096
```

Comment:
What do you think about making this public? It'd be nice to document it and allow users to inspect it.